pkgs/tools/system/swaysettings/default.nix

```
@@ -0,0 +1,75 @@
+{
+  pkgs,
```

```suggestion
{ lib
, stdenv
```

No one puts the whole `pkgs` on the scope. Instead, the packages eventually needed are listed on the input set.

---

Ok corrected